Gemfile.lock

```diff
@@ -114,54 +115,58 @@ GEM
     coffee-script-source (1.12.2)
     concurrent-ruby (1.1.9)
     crass (1.0.6)
-    dalli (2.7.11)
+    dalli (3.1.5)
```

Comment:
careful with such kind of moves, I'm pretty sure it's not a dry-* dependency, but you change a major version of it